doc/cabal-commands.rst

```diff
@@ -16,25 +15,30 @@ legacy sections. We talk in detail about some of the new-style project commands,
 
     Commands:
     [global]
+    update            Updates list of known packages.
+    install           Install packages.
     help              Help about commands.
 
     [package]
-    list-bin          list path to a single executable.
+    configure         Add extra project configuration.
+    build             Compile targets within the project.
+    clean             Clean the package store and remove temporary files.
+
+    run               Run an executable.
+    repl              Open an interactive session for the given component.
+    test              Run test-suites.
+    bench             Run benchmarks.
+
+    sdist             Generate a source distribution file (.tar.gz).
+
+    freeze            Freeze dependencies.
+    haddock           Build Haddock documentation.
+    exec              Give a command access to the store.
+    list-bin          List path to a single executable.
 
     [new-style projects (forwards-compatible aliases)]
-    v2-build          Compile targets within the project.
-    v2-configure      Add extra project configuration
-    v2-repl           Open an interactive session for the given component.
-    v2-run            Run an executable.
-    v2-test           Run test-suites
-    v2-bench          Run benchmarks
-    v2-freeze         Freeze dependencies.
-    v2-haddock        Build Haddock documentation
-    v2-exec           Give a command access to the store.
-    v2-update         Updates list of known packages.
-    v2-install        Install packages.
-    v2-clean          Clean the package store and remove temporary files.
-    v2-sdist          Generate a source distribution file (.tar.gz).
+    Since cabal-install-3.0.0.0 all v2 prefixed commands are alias for the default, unprefixed ones.
```

Comment:
"aliases"? unsure about the grammar here, but as is it reads a bit weird.

Reply:
my english is pretty bad, suggestions for rephrasing it would be very welcomed :slightly_smiling_face:

Reply:
Perhaps this? "Since cabal-install-3.0.0.0, all 'v2-' prefixed names of commands are just aliases for the simple unprefixed names."

Reply:
```suggestion
    Since cabal-install-3.0.0.0, all 'v2-' prefixed names of commands are just aliases for the simple unprefixed names.
```

Reply:
"Since cabal 3.0, all v2-prefixed commands are aliases for the default unprefixed ones."

^ That's a nicer sentence that I believes conveys the same information. Though I'm a bit unsure whether this is actually useful information or the best way to put it, don't really know what we're trying to say here.